omniwitness/logs.yaml

```diff
@@ -1,58 +1,201 @@
 Logs:
-  - Origin: go.sum database tree
```

Comment:
Accidental change detected!

Maybe we should make the witness hammer use a different file? I seem to remember that I fell into this trap (or nearly did) when I was working on this.

Reply:
`jj` caught me out on this one and the test keys one below :)